spec/lib/rmt/scc_spec.rb

```diff
@@ -327,4 +327,17 @@ def scc
   def find_predecessor_association(product_id, kind, predecessor_id)
     ProductPredecessorAssociation.find_by(product_id: product_id, kind: kind, predecessor_id: predecessor_id)
   end
+
+  describe 'Beta display' do
```

Comment:
Hm :thinking: Not sure if this is the right place to test the fix. Since nothing special really happens now in the `sync`, but what is new and needs a test is the `Product` model method `friendly_name`.
So I'd just right a test for that instead.

Reply:
I agree that would be a better place, and maybe in the product_serializer.spec.